src/core/examples.js

```diff
@@ -33,13 +49,18 @@ export function run(conf) {
   if (!examples.length) return;
 
   document.head.insertBefore(
-    hyperHTML`<style>${css}</style>`,
+    html`
+      <style>
+        ${css}
+      </style>
+    `,
```

Comment:
I don't like this but prettier now wants to to this 😕

Reply:
could be worse 👍